stdlib/private/SwiftReflectionTest/SwiftReflectionTest.swift

```diff
@@ -211,15 +211,15 @@ internal func sendBytes() {
   let count = Int(readUInt())
   debugLog("Parent requested \(count) bytes from \(address)")
   var totalBytesWritten = 0
-  var pointer = UnsafeMutableRawPointer(bitPattern: address)!
+  var pointer = UnsafeMutableRawPointer(bitPattern: address)
   while totalBytesWritten < count {
     let bytesWritten = Int(fwrite(pointer, 1, Int(count), stdout))
     fflush(stdout)
     if bytesWritten == 0 {
       printErrnoAndExit()
     }
     totalBytesWritten += bytesWritten
-    pointer = pointer.advanced(by: bytesWritten)
+    pointer = pointer?.advanced(by: bytesWritten)
```

Comment:
Check `count` for what exactly? From the code of this function it is not clear whether there are any limitations on `count` or `pointer`, and whether they are connected.

Looks like `fwrite` accepts an optional pointer just fine, and advancing a nil pointer would still fail with force unwrap.

I must be missing something...